pkg/operator/controller/ingress/controller.go

```diff
@@ -705,7 +705,21 @@ func (r *reconciler) ensureIngressController(ci *operatorv1.IngressController, d
 		errs = append(errs, err)
 	}
 
-	haveDepl, deployment, err := r.ensureRouterDeployment(ci, infraConfig, ingressConfig, apiConfig, networkConfig)
+	var haveClientCAConfigmap bool
+	clientCAConfigmap := &corev1.ConfigMap{}
+	if len(ci.Spec.ClientTLS.ClientCA.Name) != 0 {
+		clientCAConfigmapName := types.NamespacedName{
+			Namespace: "openshift-ingress",
```

Comment:
Can we use a constant here instead of `"openshift-ingress"`?

Reply:
I updated this to use the namespace from `ensureRouterNamespace`.